src/middleware/warnMutations.js

```diff
@@ -0,0 +1,54 @@
+import isEqual from 'lodash/lang/isEqual';
+import any from 'lodash/collection/any';
+import cloneDeep from 'lodash/lang/cloneDeep';
```

Comment:
A recent pull request removed lodash dependency. I think it would be better to avoid using lodash